src/Components/Patient/PatientNotes.tsx

```diff
@@ -34,6 +35,9 @@ const PatientNotes = (props: PatientNotesProps) => {
   const [reload, setReload] = useState(false);
   const [facilityName, setFacilityName] = useState("");
   const [patientName, setPatientName] = useState("");
+  const [reply_to, setReplyTo] = useState<PatientNotesModel | undefined>(
+    undefined,
+  );
```

Comment:
Shorter equivalent for the same (with naming convention corrected)

```suggestion
  const [replyTo, setReplyTo] = useState<PatientNotesModel>();
```